aspnetcore/migration/22-to-30.md

```diff
@@ -456,6 +456,29 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+## Opt in to runtime compilation of Razor views
```

Comment:
I pulled the whole section up to sit directly after **MVC service registration** as more of a logical grouping. I realise that this wasn't requested and may have been given a lot of thought to end up where it was, so do let me know and I'll put it back.